synchronizer/synchronizer.go

```diff
@@ -1143,7 +1143,7 @@ func (s *ClientSynchronizer) reorgPool(dbTx pgx.Tx) error {
 
 	// Add txs to the pool
 	for _, tx := range txs {
-		err = s.pool.StoreTx(s.ctx, *tx)
+		err = s.pool.StoreTx(s.ctx, *tx, "reorg")
```

Comment:
"reorg" as an IP? why it is needed to change the IP at this moment?

Reply:
yes, but I understand this is more related to the status than the IP. 

We just need this because we are re-adding to the pool a tx that was already added by a user before and was affected by the reorg, my suggestion is that either we save the IP to the transaction table(not recommended) or save the IP of the synchronizer machine(we can still use this information as a debug if we use a local IP instead of a public one). It's a bit weird to me to have a column that should contain an IP and find a word.

Reply:
The goal of this information is to create scripts that will ban IPs based on behaviour. I don't want the IP of the syncrhonizer to get banned by error.